docs/welcome.md

```diff
@@ -1,52 +1,51 @@
 ---
 id: welcome
-title: Getting Started
+title: Get Started
 description: Master Amplication with in-depth developer guides, API details, and tutorials.
 sidebar_label: Introduction
 slug: /
 ---
 
 # Get Started with Amplication
```

Comment:
@dericksozo - What's the goal of the change? What are the KPIs?
Let's define these- so we can better change the doc according to the goal, and track the results according to the KPIs